config/settings.d/dhcp_isc.yml.example

```diff
@@ -0,0 +1,13 @@
+---
+#:config: /etc/dhcp/dhcpd.conf
```

Comment:
The order in this file is muddled, this used to be under a "Redhat 6" (ugh) heading with the other OSes.  I'd suggest some whitespace between the config/lease sections and key/secret sections would help make it more readable.